introduction.md

```diff
@@ -1 +1,8 @@
-Short summary about myself. My favorite code school path is HTML
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?